### DISCUSSION TITLE
SDV Connectivity status

### DISCUSSION THREAD

**Discussion by antaradey-itk** [eclipse-leda/leda-distro#100] [Q&A] [answered]
After completion of device provisioning using sdv-provision, I get the status of Cloud Connector and Device ID under SDV Connectivity as failed, while the Kanto CM Containers are running and OK. Is this the current state of the SW or is there some issue? 
I could not find anything specific regarding this in the documentation of Leda. 
I want to continue with launching an example vehicle application.

![sdv-health_](https://user-images.githubusercontent.com/129940848/236410009-40cb9a10-2559-471c-8517-fb92d4d6d213.png)

**Comment by mikehaller** [comment, marked as answer]
Indeed you reached the current state of the software: the device provisioning part has not been fully implemented yet and lacks proper configuration and documentation.

We currently have multiple ways and cloud backends:

- **Leda Cloud Connector:** can connect to a remote Azure IoT Hub. Requires manual configuration as minimally described in the Leda documentation or in the `sdv-provision` script. You would need to continue the device provisioning process and configure the cloud connector to make this work. 
- **Kanto Azure Connector:** could be used, but not yet integrated or tested with Leda.
- **Kanto Suite Connector:** could be used, but not yet integrated or tested with Leda.
- **Eclipse BfB:** this is a standardization project and aims to publish a specification for the edge-cloud protocol. As the project has not yet published any contents, nobody started an open source implementation yet (neither on edge side, nor on cloud side) afaik.

As for now, you should be able to run the vehicle applications without the cloud connection, as the cloud connection is supposed to be optional and currently mainly used for update triggers.

**Reply by kowshikk2** [reply]
Can you elaborate what you mean by manual configuration in Leda Cloud Connector. What steps need to be done?
